backend/inspirehep/migrator/cli.py

```diff
@@ -120,14 +120,23 @@ def migrate_file(file_name, mirror_only=False, force=False, wait=False):
 @click.option(
     "-w", "--wait", is_flag=True, default=False, help="Wait for all subtasks to finish."
 )
+@click.option(
+    "-d",
+    "--date-from",
+    "date_from",
+    default=None,
+    help="Date from which records should be migrated. (YYYY-MM-DD)",
+)
```

Comment:
looks like `click` 7 has automatic datetime conversion, but we're still on 6.* :cry: